### PR TITLE
fixes in wallet

### PR DESCRIPTION
**Pull request by KatarzynaStefanowiczPagepro**
## Description

- [] TODO write description

**New stuff** ✨

* New stuff goes here

**Changes** 🏗

* Changes go here

**Deletions** ⚰️

* Deletions go here

## TODO

- [ ] Add TODOs

(Resolves | Contributes to) #31415

### CODE REVIEW

**Comment by KatarzynaStefanowiczPagepro** [comment]
The bunch of fixes in wallet connected menu
image

https://tw.auroracreation.com/app/tasks/15516310
https://tw.auroracreation.com/app/tasks/15516332
https://tw.auroracreation.com/app/tasks/15516331
https://tw.auroracreation.com/app/tasks/15516335
https://tw.auroracreation.com/app/tasks/15516471

File: src/components/common/Extensions/UserHub/partials/StakesTab/partials/StakesTabItem.tsx [hunk 1]
```diff
@@ -15,10 +17,15 @@ const StakesItems: FC<StakesProps> = ({
   status,
 }) => {
   return (
-    <li className="flex flex-col py-4 border-b-[0.0625rem] border-gray-100">
+    <li
+      className={clsx(
+        styles.stakesItem,
+        'first:pt-2 last:pb-6 sm:first:pt-0 sm:last:pb-1.5',
```
Comment (CzarekDryl) [inline src/components/common/Extensions/UserHub/partials/StakesTab/partials/StakesTabItem.tsx:23]: Can't this styles be also in `StakesTabItem.module.css`?

Comment (KatarzynaStefanowiczPagepro) [reply src/components/common/Extensions/UserHub/partials/StakesTab/partials/StakesTabItem.tsx]: Moved

File: src/components/common/Extensions/UserHub/partials/StakesTab/StakesTab.tsx [hunk 2]
```diff
@@ -35,9 +35,26 @@ const StakesTab = () => {
...
 
+  const claimedNotificationNumber = useMemo(
+    () => stakes.filter(({ status }) => status === 'claimed').length,
+    [stakes],
+  );
+
+  const updatedTabsItems = useMemo(
+    () =>
+      tabsItems.map((item) => {
+        return item.title === 'Claimable'
```
Comment (krawczykkatarzyna) [inline src/components/common/Extensions/UserHub/partials/StakesTab/StakesTab.tsx:46]: maybe better will be filtering by some id?

File: src/components/common/Extensions/UserHub/partials/TransactionsTab/partials/GroupedTransactionContent.tsx [hunk 3]
```diff
@@ -73,11 +73,11 @@ const GroupedTransactionContent: FC<GroupedTransactionContentProps> = ({
         'before:bg-success-400': ready || succeeded,
         'before:bg-negative-400': failed,
         'before:bg-blue-400': pending,
-        'font-bold': selected,
+        'font-semibold': selected,
       })}
     >
       <div className="flex justify-between items-center">
-        <h4>
+        <h4 className="text-gray-900">
```
Comment (krawczykkatarzyna) [inline src/components/common/Extensions/UserHub/partials/TransactionsTab/partials/GroupedTransactionContent.tsx:80]: is text-gray-900 necessary?

Comment (KatarzynaStefanowiczPagepro) [reply src/components/common/Extensions/UserHub/partials/TransactionsTab/partials/GroupedTransactionContent.tsx:80]: Yes, it overrides the styles added by default to popover

File: src/components/common/Extensions/UserHub/partials/StakesTab/StakesTab.tsx (hunk 2 above)
Comment (KatarzynaStefanowiczPagepro) [reply src/components/common/Extensions/UserHub/partials/StakesTab/StakesTab.tsx:46]: tabsItems looks like:
```
export const tabsItems: TabItem[] = [
  { id: 0, title: 'All' },
  { id: 1, title: 'Finalized' },
  { id: 2, title: 'Claimable' },
];
```
and are typed by TabsItem[] interface.
How I could do it better?

Comment (KatarzynaStefanowiczPagepro) [reply src/components/common/Extensions/UserHub/partials/StakesTab/StakesTab.tsx]: I've added a type prop

**Review by krawczykkatarzyna — APPROVED** [review]

**Pull request merged**